recipe/meta.yaml

```diff
@@ -59,9 +59,8 @@ requirements:
     - xattr >=1.0.0,<2.0.0    # [osx]
```

Comment:
Think the challenge with using `__unix` here is this dependency will either install on macOS and Linux or it will be ignored altogether

Instead would suggest ignoring the linter and sticking with what you had before